internal/lib/container_server.go

```diff
@@ -769,3 +784,19 @@ func (c *ContainerServer) UpdateContainerLinuxResources(ctr *oci.Container, reso
 
 	c.state.containers.Add(ctr.ID(), ctr)
 }
+
+func shutdownWasUnclean(config *libconfig.Config) bool {
```

Comment:
instead of just copy-pasting, can you export this function somewhere and import it in both places? Ideally export here because I think internal/criocli already needs to import this package

Reply:
@haircommander done 👍🏽